js/src/views/Settings/Background/background.js

```diff
@@ -56,15 +57,29 @@ class Background extends Component {
   render () {
     return (
       <Container>
-        <ContainerTitle title='Background Pattern' />
+        <ContainerTitle title={
+          <FormattedMessage id='settings.background.label' />
+        } />
         <div className={ layout.layout }>
           <div className={ layout.overview }>
-            <div>The background pattern you can see right now is unique to your Parity installation. It will change every time you create a new Signer token. This is so that decentralized applications cannot pretend to be trustworthy.</div>
-            <div>Pick a pattern you like and memorize it. This Pattern will always be shown from now on, unless you clear your browser cache or use a new Signer token.</div>
+            <div>
+              <FormattedMessage
+                id='settings.background.overview_0'
+                defaultMessage='The background pattern you can see right now is unique to your Parity installation. It will change every time you create a new Signer token. This is so that decentralized applications cannot pretend to be trustworthy.' />
```

Comment:
If we have a `defaultMessage` (which I think is good for readability), we do we need a separate `en` i18n file? They will probably not stay consistent over time.